src/Build.UnitTests/BuildEventArgsWriter_Tests.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.IO;
+using Xunit;
+using Microsoft.Build.Logging;
+using Microsoft.Build.Shared;
+
+namespace Microsoft.Build.Engine.UnitTests
+{
+    public class BuildEventArgsWriter_Tests
+    {
+        [Fact]
+        public void WriteBlobFromStream()
```

Comment:
`src\Build.UnitTests\BuildEventArgsSerialization_Tests.cs` already contains round trip test cases for `BuildEventArgsWriter`. I wonder if it's a better home for the new test.

Reply:
Sure, this class looks like better home for the test. Moved.